lib/src/wrappers/ResponsiveWrapper.tsx

```diff
@@ -1,27 +1,27 @@
 import * as React from 'react';
 import useMediaQuery from '@material-ui/core/useMediaQuery';
-import { useTheme } from '@material-ui/core/styles';
+import { IS_TOUCH_DEVICE_MEDIA } from '../constants/dimensions';
 import { MobileWrapperProps, MobileWrapper } from './MobileWrapper';
 import { DesktopWrapperProps, DesktopWrapper } from './DesktopWrapper';
-import { Breakpoint } from '@material-ui/core/styles/createBreakpoints';
 import { DesktopPopperWrapperProps, DesktopPopperWrapper } from './DesktopPopperWrapper';
 
 export interface ResponsiveWrapperProps
   extends DesktopWrapperProps,
     DesktopPopperWrapperProps,
     MobileWrapperProps {
-  /** Breakpoint when `Desktop` mode will be changed to `Mobile`
-   * @default 'md'
+  /** Css media query when `Mobile` mode will be changed to `Desktop`
+   * @default "@media (pointer: fine)"
+   * @example "@media (min-width: 720px)" or theme.breakpoints.up("sm")
    */
-  desktopModeBreakpoint?: Breakpoint;
+  desktopModeMediaQuery?: string;
```

Comment:
Have you considered a shorter name for the prop?

```suggestion
  desktopMediaQuery?: string;
```

Reply:
I think the shorter variant is less expressive. What "desktop" means?

Reply:
It could be interpreted as "a media query for desktop", definitely not as good as the information we can provide in a description. Adding mode definitely provide extra context. I don't know. I was wondering about this tradeoff of information vs length for the props, e.g. `openPickerIcon`, vs `pickerIcon`.